tools/generate-helpers/pg-table-bindings/main.go

```diff
@@ -204,6 +207,7 @@ func main() {
 	c.Flags().StringVar(&props.SchemaDirectory, "schema-directory", "", "the directory in which to generate the schema")
 	c.Flags().BoolVar(&props.SingletonStore, "singleton", false, "indicates that we should just generate the singleton store")
 	c.Flags().StringSliceVar(&props.SearchScope, "search-scope", []string{}, "if set, the search is scoped to specified search categories. comma seperated of search categories")
+	c.Flags().BoolVar(&props.CachedStore, "cached-store", false, "if true, ensure the store is mirrored in a memory cache")
```

Comment:
WDYT about adding a "hint" that this should only be used on lower-cardinality stores and after careful consideration / good reason due to its added memory pressure?

Something similar to the comment we have on the `GetAll` function in the generic store, but OTOH I suppose the only way would be a comment within the generated code _if_ the cache is enabled (since most people won't look at the flags definitions here).